File: snapshots/input/syntax/Main/Properties.cs
using System.Diagnostics.CodeAnalysis;

namespace Main;

[SuppressMessage("ReSharper", "all")]
public class Properties
{
    byte Get { get; }

    char Set
    {
        set { throw new NotImplementedException(); }
    }

    uint GetSet { get; set; }
    long SetGet { set; get; }

    string? Init { get; init; }
}

File: snapshots/input/syntax/VBMain/Properties.vb
Imports System.Diagnostics.CodeAnalysis

Namespace VBMain
    <SuppressMessage("ReSharper", "all")>
    Public Class Properties
        Private ReadOnly Property [Get] As Byte

        Private WriteOnly Property [Set] As Char
            Set(ByVal value As Char)
                Throw New NotImplementedException()
            End Set
        End Property

        Private Property GetSet As UInteger
        Private Property SetGet As Long
    End Class
End Namespace
